controllers/customer-admin-controller/build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM alpine:3.6
```

Comment:
s/alpine/centos/

Reply:
I haven't actually built this but that definitely makes sense.